packages/i18n/data/core.json

```diff
@@ -40,8 +40,7 @@
   "FailedAuthentication.title": "We are unable to authorize you to use the Merchant Center.",
   "NavBar.MCSupport.title": "Support",
   "Notification.hideNotification": "Hide notification",
-  "PageNotFound.helpDesk": "Help Desk",
-  "PageNotFound.paragraph1": "The item you are looking for may have been deleted, does not exist, or the URL was entered incorrectly. Check the URL and try again.\n\nPlease contact your system administrator or the commercetools {link} if you have any further questions.",
+  "PageNotFound.paragraph1": "The item you are looking for may have been deleted, does not exist, or the URL was entered incorrectly. Check the URL and try again.\n\nPlease contact your system administrator or the commercetools <a>Help Desk</a> if you have any further questions.",
```

Comment:
I'll manually update the messages in TX after merging this, to reflect the new changes